README.md

```diff
@@ -968,7 +949,7 @@ nock.cleanAll()
 You can make all the interceptors for a scope persist by calling `.persist()` on it:
 
 ```js
-var scope = nock('http://persisssists.con')
+const scope = nock('http://persisssists.con')
```

Comment:
Guessing this is just playing with words here, but we should change it to .com instead. :)

Reply:
I would suggest to use `https://example.test`?

Reply:
I made it `example.com` since usually our users will be intercepting requests for real hosts.